standard/flags.lua

```diff
@@ -0,0 +1,123 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Flags
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local FlagData = mw.loadData('Module:Flags/MasterData')
+
+local Flags = {}
+
+-- Template functions
+function Flags.Flag(frame)
+	return Flags._Flag(frame.args[1])
+end
+function Flags.FlagNoLink(frame)
+	return Flags._FlagNoLink(frame.args[1])
+end
+function Flags.CountryName(frame)
+	return Flags._CountryName(frame.args[1])
+end
```

Comment:
As a temporary step we probably should change this to
```suggestion
-- Template functions
function Flags.Flag(frame)
	if type(frame) == 'string' then
		Flags._Flag(frame)
	end
	return Flags._Flag(frame.args[1])
end
function Flags.FlagNoLink(frame)
	if type(frame) == 'string' then
		Flags._FlagNoLink(frame)
	end
	return Flags._FlagNoLink(frame.args[1])
end
function Flags.CountryName(frame)
	if type(frame) == 'string' then
		Flags._CountryName(frame)
	end
	return Flags._CountryName(frame.args[1])
end
```
that would allow us to change it in the modules (here on the git, on commons and on all the wikis, so would take quite a bit of work)
after that is done we could change it to using `Module:Class`